storage-client/src/client.rs

```diff
@@ -1,20 +1,21 @@
 use anyhow::{anyhow, Ok, Result};
-use arrow_array::RecordBatch;
+use arrow::array::RecordBatch;
 use futures::stream::StreamExt;
-use futures::TryStreamExt;
+
 use hyper::Uri;
 use lazy_static::lazy_static;
-use parquet::arrow::{ParquetRecordBatchStreamBuilder, ProjectionMask};
+use parquet::arrow::arrow_reader::ParquetRecordBatchReaderBuilder;
```

Comment:
Difference?

Reply:
We have to switch to arrow `50.0.0` since `istziio_client` uses `50.0.0`. And we also need to use `arrow::record_batch::RecordBatch` since `istziio_client` uses this one. Some changes are made accordingly including this.